Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.datadrivendiscovery.org/jpl/docker_images/complete:ubuntu-artful-python36-v2019.2.18-20190303-060946
-ADD $EXPERIMENTER /d3m-experimenter
+FROM registry.datadrivendiscovery.org/jpl/docker_images/complete:ubuntu-bionic-python36-v2019.6.7
+ADD . /d3m-experimenter
```

Comment:
It looks like I made this change @orionw, should I have done that? The change from `$EXPERIMENTER` TO `.`?

Reply:
Are you sure you did?  I thought it was me.  Yeah it should work fine, since `.` is the location of the experimenter

Reply:
Should be good to merge if you're okay with that.  If not, feel free to change it